### DISCUSSION TITLE
DataDog metrics only with DataDog APM enabled?

### DISCUSSION THREAD

**Discussion by nightcoding** [imgproxy/imgproxy#1295] [Q&A]
Hello there 👋🏻 

We succesfully deployed the image proxy in our k8s cluster on GCP. We also do have datadog, and the datadog agents are running in the cluster. I double checked and dogstatd is enabled. We do not have the APM package, so this is not enabled and port 8126 is not open for connections. Port `8125` is up and running.

So we tried out the datadog integration of the imgproxy and there are no metrics shown in datadog. I can't find any metrics starting with `imgproxy.*`.

I see the following error in the image proxy:

```
INFO    [2024-05-27T07:09:31Z] Datadog Tracer v1.63.1 ERROR: Decoding features: EOF (occurred: 27 May 24 07:08 UTC) 
```

Is it possible that the image proxy will only send metrics if the datadog APM package is available?

Thanks for any help!
Martin

**Comment by DarthSim** [comment]
Hey @nightcoding!
The error seems to be happening here: https://github.com/DataDog/dd-trace-go/blob/b0aa1b8aa856084eed0abb1957719284319130d6/ddtrace/tracer/option.go#L624. Probably, something is wrong with your DD agent. I'd check its logs.
